components/ListItem.js

```diff
@@ -0,0 +1,16 @@
+import styled from "styled-components";
+
+export default function ListItem({ name, isChecked }) {
```

Comment:
This prop called name is confusing, is it really a name or the text of the list item?